lib/components/viewers/stop-viewer.js

```diff
@@ -158,20 +244,25 @@ class RouteRow extends Component {
   }
 
   render () {
-    const { route, stopTimes, homeTimezone } = this.props
+    const { pattern, stopTimes, homeTimezone, stopViewerArriving } = this.props
+    const { route } = pattern
 
     // sort stop times by next departure
     let sortedStopTimes = null
     if (stopTimes) {
       sortedStopTimes = stopTimes.sort((a, b) => {
-        const aTime = a.serviceDay + a.realtimeDeparture
-        const bTime = b.serviceDay + b.realtimeDeparture
+        const aTime = a.serviceDay + a.realtimeArrival
+        const bTime = b.serviceDay + b.realtimeArrival
```

Comment:
Why the change to arrival? Also, previous comments about implementing sort by subtracting numbers apply here.

Reply:
Actually, this appears to be a mistake. I thought I remember seeing something in [the ticket](https://github.com/ibi-group/trimet-mod-otp/issues/184) about changing from departures to arrivals, but I must be mistaken. I will revert back to departures.